ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+# Table of Contents <!-- omit in toc -->
+- [Ansible and AVD Overview](#ansible-and-avd-overview)
+  - [Day 1 operation i.e. provisioning a network from scratch](#day-1-operation-ie-provisioning-a-network-from-scratch)
+  - [What is Ansible](#what-is-ansible)
+  - [What is required to run Ansible](#what-is-required-to-run-ansible)
+  - [What is Ansible AVD (Ansible collection for Arista Validated Designs)](#what-is-ansible-avd-ansible-collection-for-arista-validated-designs)
+  - [When to use Ansible AVD and when not to](#when-to-use-ansible-avd-and-when-not-to)
+  - [Change your mindset and culture](#change-your-mindset-and-culture)
+  - [How does Ansible work](#how-does-ansible-work)
+    - [Inventories](#inventories)
+    - [Organizing host and group variables](#organizing-host-and-group-variables)
+    - [Group Variables](#group-variables)
+    - [Playbooks](#playbooks)
+  - [Source of Truth](#source-of-truth)
+  - [How do I use Ansible AVD](#how-do-i-use-ansible-avd)
+  - [Day 2 Operations](#day-2-operations)
+
+# Ansible and AVD Overview
+## Day 1 operation i.e. provisioning a network from scratch
+Imagine being tasked with installing and configuring a leaf-spine network with 2 spine switches and 4 leaf switches from scratch:
+
+![Figure: Arista Leaf Spine topology](../_media/getting-started/LS-Topology.png)
+
+Traditionally, one would grab a laptop, a console cable and perhaps a USB key containing the Arista EOS image of choice.
+This would mean a tedious device by device process of performing a software upgrade and typing configuration commands manually.
+Alternatively by using pre-developed config snippets based on reading the relevant
+Arista design guides and/or previous experience *(did anyone think of copy/paste from existing devices!?)*.
+The configurations would be manually adapted per device, which is time consuming.
+The operator could easily become burned out, leading to user error, impacting the network.
+
+Using Ansible collection for Arista Validated Designs (AVD), such a task is automated, greatly simplified and many times faster,
+where all the “basic” network configuration is guaranteed to be identical across the entire fabric from day one.
+No longer do you have to manually inspect each and every device for errors and this also applies when updating the
+configuration in day 2 operations. In addition, you do not have to spend any time integrating the Arista Design and Deployment Guides,
+since Ansible AVD is built on those as a foundation. Ansible AVD also fully integrates with Arista Cloudvision,
```

Comment:
Ansible-AVD, keep it one word to make sure we don't confuse users for taking it as 2 entities (Ansible and AVD)

Also rephrase to ```since Ansible-AVD is built on those principles.```